monkey/infection_monkey/post_breach/actions/modify_shell_startup_files.py

```diff
@@ -1,8 +1,13 @@
+import subprocess
+
 from common.data.post_breach_consts import \
     POST_BREACH_SHELL_STARTUP_FILE_MODIFICATION
 from infection_monkey.post_breach.pba import PBA
 from infection_monkey.post_breach.shell_startup_files.shell_startup_files_modification import \
     get_commands_to_modify_shell_startup_files
+from infection_monkey.telemetry.post_breach_telem import PostBreachTelem
+
+EXECUTION_WITHOUT_OUTPUT = "(PBA execution produced no output)"
 
 
 class ModifyShellStartupFiles(PBA):
```

Comment:
Not mandatory, but general rule is 1 class per 1 file. If you think that one class is only relevant to another, you can nest them inside.